### PR TITLE
Less horrible old things

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
Laser guns now when old can cost more to fire, have less in their charge mode and charge less fast in charge mode!
Tools and guns now will not be as horrable if old, so no longer will we have 10% -20% tools

## Changelog
:cl:
/:cl:

### CODE REVIEW

File: code/modules/scrap/oldificator.dm [hunk 1]
```diff
@@ -60,17 +60,58 @@
...
+	..()
+
+/obj/item/weapon/gun/energy/make_old()
+	. = ..()
+	charge_cost+= rand(0,250)
+	overcharge_max-= rand(0,5) //This is infact a number you want to go up
+	overcharge_rate-= rand(0,5)
+
+/obj/item/weapon/gun/energ/make_young()
+	if(oldified)
```
Comment (KathyRyals) [inline code/modules/scrap/oldificator.dm:103]: Early returns.

File: code/modules/scrap/oldificator.dm [hunk 2]
```diff
@@ -60,17 +60,58 @@
...
 	fire_delay+= rand(0,3)
-	recoil_buildup+= rand(0,20)
-	damage_multiplier = damage_multiplier*(rand(1,10)/10)
-	penetration_multiplier = penetration_multiplier*(rand(1,10)/10)
+	recoil_buildup+= rand(0,10)
+	damage_multiplier = damage_multiplier*(rand(8,10)/10) //20% less damage max
+	penetration_multiplier = penetration_multiplier*(rand(8,10)/10) //20% less damage penetration
+
+/obj/item/weapon/gun/make_young()
+	if(oldified)
```
Comment (KathyRyals) [inline code/modules/scrap/oldificator.dm:86]: Use early returns.

**Review by KathyRyals — CHANGES_REQUESTED** [review]
Should probably make sure to remove the duplicates.

File: code/modules/scrap/oldificator.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/scrap/oldificator.dm:103]: This is checking if it even is old in the first place. Its how clothing is coded, and I know that works

Comment (KathyRyals) [reply code/modules/scrap/oldificator.dm:103]: I know what it does, and I'm telling ya to use early returns for the sake of making good code.

**Review by KathyRyals — APPROVED** [review]
Lookin' good.

**Pull request merged**